### DISCUSSION TITLE
Deploying tauri

### DISCUSSION THREAD

**Discussion by rdwnsjjd** [tauri-apps/tauri#3028] [Q&A] [answered]
How to ~~deploy~~ (build) Tauri app with react?
is there a guide?

**Comment by amrbashir** [comment, marked as answer]
What do you mean by deploy? have you checked `npx create-tauri-app` ?

**Reply by rdwnsjjd** [reply]
I finished my development and want to build the final exe file (in Linux).
sorry, in fact, my question is how to build a Tauri app with react

**Reply by amrbashir** [reply]
run `tauri build` to build the final executable on the platform you're using, if you want to build for other platforms, then switch to that platform and build on it, or use a VM or a docker container or better use github action with [tauri-action](https://github.com/tauri-apps/tauri-action).

**Reply by amrbashir** [reply]
Just a friendly note, your recent questions are better suited to be asked in our Discord server.

**Reply by rdwnsjjd** [reply]
Unfortunately, I don't have access to discord.
this solution is for building tauri. OK. is it same when I use react? Shouldn't I do more?
At the moment I built my app by `tauri build` command. but when I run final exe file, it shows a blank white window with a sentense: 
```
Could not get requested file.
```
while react is started

**Reply by amrbashir** [reply]
> Unfortunately, I don't have access to discord. 

That's unfortunate. It is fine to use discussion Q&A then.

For any framework you use with tauri, you need to build it first before building tauri iteself.
In react for example, you need to do: 
```sh
yarn build # this build react app into "dist" folder I think
yarn tauri build # this builds the final executable but make sure that tauri.conf.json has `distDir` set to the same folder that the previous command dist folder
```

**Reply by rdwnsjjd** [reply]
let me check...

**Reply by rdwnsjjd** [reply]
successful!
tnx

**Reply by amrbashir** [reply]
great! 

I highly recommend you create a discord account, and join our server, we and the community can help faster there with these types of question.

**Reply by rdwnsjjd** [reply]
OK, I will try it.
tnx
